src/components/pages/order/Main/Admin/AdminTabs.js

```diff
@@ -22,7 +22,7 @@ function AdminTabs() {
   };
 
   // config file method!!!
-  const tabs = tabsConfig;
+  const tabs = getTabsConfig();
```

Comment:
```suggestion
  const tabs = tabsConfig;
```

car ta fonction ne prend aucun paramètre. Quand elle prendra des paramètres, **là** tu transformeras ta variable en fonction en la renommant **getTabsConfig()**.